opapi/src/generators/errors.ts

```diff
@@ -88,25 +101,26 @@ export const errorFrom = (err: unknown): ApiError => {
 }
 
 function getErrorFromObject(err: object) {
```

Comment:
Yes, however this will end up showing "An invalid error occurred" for all non-ApiError's, as `errorFrom()` is used as fallback in the error handlers, so we still need these lines:

```ts
if (err instanceof Error) {
  return new UnknownError(err.message, err)
}
```

I like the simplification so I've refactored this code a little bit here:
https://github.com/botpress/packages/pull/42/commits/580a247d4f2bd18933257f5a9304b500beaf7927